### PR TITLE
Merge 5.3.1 release branch back into `master`

### PR DESCRIPTION
**Pull request by str4d**

### CODE REVIEW

**Comment by str4d** [comment]
Opened this PR to get CI running on it, but we should ensure that release binaries can be built before merging this PR.

**Comment by str4d** [comment]
Release binaries have been built; this can be merged after review.

**Review by sellout — APPROVED** [review]
utACK

**Review by nuttycom — APPROVED** [review]
utACK

**Pull request merged**
